de_web_plugin.cpp

```diff
@@ -10606,11 +10719,11 @@ bool DeRestPluginPrivate::processZclAttributes(Sensor *sensorNode)
 
             if (writeAttribute(sensorNode, sensorNode->fingerPrint().endpoint, BASIC_CLUSTER_ID, attr, VENDOR_PHILIPS))
             {
-                ResourceItem *item = sensorNode->item(RConfigPending);
-                uint8_t mask = item->toNumber();
-                mask &= ~R_PENDING_LEDINDICATION;
-                item->setValue(mask);
-                sensorNode->clearRead(WRITE_LEDINDICATION);
+                // ResourceItem *item = sensorNode->item(RConfigPending);
+                // quint16 mask = item->toNumber();
+                // mask &= ~R_PENDING_LEDINDICATION;
+                // item->setValue(mask);
+                // sensorNode->clearRead(WRITE_LEDINDICATION);
```

Comment:
These relate to "Clear pending on receiving the attribute report, instead of on queueing the Write Attributes;"?

I think the commented out lines can be removed in further PRs to nix some dead code.

Reply:
Yes, just being over careful.  And as I mentioned, _Occupied to Unoccupied Delay_ isn’t reportable, so I kept the code for that attribute.  Would really want to clear the `pending` on the _Write Attributes Response_, but I don’t think the API plugin handles these.